### DISCUSSION TITLE
New to tinacms and your 'Getting started with a starter' is throwing me a EEXIST error.

### DISCUSSION THREAD

**Discussion by 96312** [tinacms/tinacms#3100] [Help]
I'm completely new to tinacms and am following your page : [https://tina.io/docs/introduction/using-starter/](https://tina.io/docs/introduction/using-starter/) by using `npx create-tina-app@latest`. I can not progress since the system is **throwing me an error on the installation step itself.** Is this a known bug?

---

```
Compilation failed with errors. Server has not been restarted. see error below
EEXIST: file already exists, mkdir '..\.tina\__generated__'
[Error: EEXIST: file already exists, mkdir '..\.tina\__generated__'] {
  errno: -4075,
  code: 'EEXIST',
  syscall: 'mkdir',
  path: ..\\.tina\\__generated__'
}
```

**Comment by jamespohalloran** [comment]
I'm not able to reproduce. Are you on a Windows machine?

**Reply by 96312** [reply]
@jamespohalloran That is correct, I'm on the Windows 10 21H1 update with Node v16.17.0.

**Reply by jamespohalloran** [reply]
I don't have a Windows machine to test, but could you try `npx create-tina-app@windows-dir-fix` and see if it fixes your issue?
(PR: https://github.com/tinacms/tinacms/pull/3152)

**Reply by 96312** [reply]
Can confirm, PR #3152 has fixed the issue on my Windows setup. I was able to get the development server up and running. Thanks for the fix.

![image](https://user-images.githubusercontent.com/34262180/189148217-7f78ad61-0023-482d-81a2-94d5e1edb529.png)

**Reply by jamespohalloran** [reply]
Great to hear! That's been merged into main so it will be part of the next release.
